en/products/relay/landing.ftl

```diff
@@ -0,0 +1,57 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/products/relay/
+
+meta-title = { -brand-name-firefox-relay }
+meta-description-2 = { -brand-name-firefox-relay } makes it easy to create email masks that forward your messages to your true inbox. Use them to protect your online accounts from hackers and unwanted messages.
+
+## HERO SECTION
+
+hero-section-title = Protect your identity with secure phone and email masking
+hero-section-body = Our secure, easy-to-use email and phone masks help keep your identity
+    private so you can sign up for new accounts anonymously, stop spam texts and junk calls, and
+    get only the emails you want in your inbox.
+# Context: This lists the various websites and magazines who have mentioned Firefox Relay.
+# Example: "As seen in: FORBES magainze and LifeHacker"
+hero-section-social-proof = As seen in:
+
+## HOW IT WORKS SECTION
+
+how-it-works-section-title = How { -brand-name-relay } email & phone masking works
+
+how-it-works-section-extension-headline = Use { -brand-name-relay } email masks and phone masks everywhere
+how-it-works-section-extension-body = Get secure, random email masks and a unique, masked phone number to use any time a website, app, store, or restaurant asks for your information.
+
+how-it-works-section-forward-headline = We’ll forward emails, phone calls, and texts to you
+how-it-works-section-forward-body = Rest assured, senders will never know your real email address or real number. You can even reply to texts and emails without sharing your real identity.
+
+how-it-works-section-manage-headline = Manage your masks from your { -brand-name-relay } dashboard
+how-it-works-section-manage-body = Sign in to your { -brand-name-relay } dashboard to create new masks, label existing masks, and  block calls, texts, or emails from masks that get too much spam.
+
+## REVIEWS SECTION
+
+landing-reviews-title = Reviews
+landing-reviews-logo-title = { -brand-name-firefox-browser }
+landing-reviews-add-ons = Add-ons
+# Do not localize addons.mozilla.org
+landing-reviews-details-source = Source: addons.mozilla.org
+# This string is displayed in a smaller font under a big number representing the average review score (e.g. "4.2").
+# Review count will stand for the total number of reviews at a given time (e.g. "328 reviews").
+landing-reviews-rating = out of 5 ({ $review_count } reviews)
```

Comment:
This should be a plural string.

Reply:
Thanks for catching this, agreed.

Reply:
Is there anything I can do to localize the display of the number as well? `1,259` vs `1 259` etc.

If not I can use a helper on the bedrock side.

Reply:
My understanding is that Fluent automatically formats numbers passed into variables. 
https://github.com/projectfluent/fluent/blob/feffc720fa5d5621a703f42731d709e216fc6730/guide/variables.md

Reply:
I will update it to make sure it's passed as an `int` not a `string`.